packages/storybook/src/stories/grid-rows.stories.tsx

```diff
@@ -629,3 +629,18 @@ export function CommodityEdit() {
     </React.Fragment>
   );
 }
+
+export function RegressionEditCell() {
```

Comment:
it could also be a suffix as well

- EditCellSnap?

Note: It seems more like a snapshot test 🤔